react-components/src/components/Checkout/__test__/__snapshots__/editableForm.test.js.snap

```diff
@@ -1,3 +1,343 @@
+          </span>
+          <p
+            class="root"
+          />
+        </div>
+      </div>
+      <div
+        class="region_code"
+      >
+        <div
+          class="root"
+        >
+          <span
+            class="label"
+          >
+            State
+          </span>
+          <span
+            class="root"
+          >
+            <span
+              class="input"
+            >
+              <input
+                class="input"
+                id="region_code"
+                name="region_code"
+                value=""
+              />
+            </span>
+            <span
+              class="before"
+            />
+            <span
+              class="after"
+            />
+          </span>
+          <p
+            class="root"
+          />
+        </div>
+      </div>
+      <div
+        class="postcode"
+      >
+        <div
+          class="root"
+        >
+          <span
+            class="label"
+          >
+            ZIP
+          </span>
+          <span
+            class="root"
+          >
+            <span
+              class="input"
+            >
+              <input
+                class="input"
+                id="postcode"
+                name="postcode"
+                value=""
+              />
+            </span>
+            <span
+              class="before"
+            />
+            <span
+              class="after"
+            />
+          </span>
+          <p
+            class="root"
+          />
+        </div>
+      </div>
+      <div
+        class="telephone"
+      >
+        <div
+          class="root"
+        >
+          <span
+            class="label"
+          >
+            Phone
+          </span>
+          <span
+            class="root"
+          >
+            <span
+              class="input"
+            >
+              <input
+                class="input"
+                id="telephone"
+                name="telephone"
+                value=""
+              />
+            </span>
+            <span
+              class="before"
+            />
+            <span
+              class="after"
+            />
+          </span>
+          <p
+            class="root"
+          />
+        </div>
+      </div>
+      <div
+        class="validation"
+      />
+      <div
+        class="error"
+      />
+    </div>
+    <div
+      class="footer"
+    >
+      <button
+        class="root_normalPriority"
+        type="button"
+      >
+        <span
+          class="content"
+        >
+          Cancel
+        </span>
+      </button>
+      <button
+        class="root_highPriority"
+        type="submit"
+      >
+        <span
+          class="content"
+        >
+          Use Address
+        </span>
+      </button>
+    </div>
+  </form>
+</DocumentFragment>
+`;
+
+exports[`<EditableForm /> should render ShippingForm if user is entering shipping information 1`] = `<DocumentFragment />`;
```

Comment:
Why is this empty? Shouldn't it display the `ShippingForm` component?

Reply:
You're right. Taking a second look.